### PR TITLE
Updates to evolution

### PR DESCRIPTION
**Pull request by yusha-sun**
### Description of the proposed additions or changes

- made some syntax/wording changes
- updated various VOC status given new data
- added some references regarding variants

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/09.evolution.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ In particular, the first case highlighted the role of convalescent plasma in cre
 In fact, one study [@doi:10.1101/2020.12.28.424451] exposed the SARS-CoV-2 virus to convalescent plasma _in vitro_ repeatedly to see how much plasma was required to neutralize the virus.
 The results of the first six exposures were similar, but they reported that after the seventh exposure (on day 45), the amount of plasma required began to increase.
 In analyzing the viral variants present, they found that this viral escape was promoted by the sudden accumulation of mutations, especially in the receptor-binding domain (RBD) and N-terminal domain (NTD), that quickly rose in frequency.
-By the thirteenth exposure (day 85), the virus had evolved three mutations and could not longer be neutralized by the plasma used.
+By the thirteenth exposure (day 85), the virus had evolved three mutations and could not longer be neutralized by the plasma used, even though the plasma comprised of polyclonal serum that targeted a variety of epitopes.
```
Comment (agitter) [inline content/09.evolution.md:101]:
```suggestion
By the thirteenth exposure (day 85), the virus had evolved three mutations and could not longer be neutralized by the plasma used, even though the plasma was comprised of polyclonal serum that targeted a variety of epitopes.
```
Does this read better?

File: content/09.evolution.md [hunk 2]
```diff
@@ -126,31 +126,34 @@ Since then, B.1.1.7 has spread across the UK and internationally, and it has now
...
 The FDA has highlighted that the performance of three diagnostic tests may be affected by the B.1.1.7 lineage because it could cause false negative tests [@url:https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2].
 
-While B.1.1.7 is currently the main VOC, other genetic variants not designated VOCs have been detected, including B.1.351 and P.1, both of which emerged independently [@url:https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html].
+While B.1.1.7 is currently the main VOC, other genetic variants also currently designated as VOCs have been detected, including B.1.351 and P.1, both of which emerged independently [@url:https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html; https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/variant-surveillance/variant-info.html].
```
Comment (agitter) [inline content/09.evolution.md:134]:
```suggestion
While B.1.1.7 is currently the main VOC, other genetic variants also currently designated as VOCs have been detected, including B.1.351 and P.1, both of which emerged independently [@url:https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html; @url:https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/variant-surveillance/variant-info.html].
```

File: content/09.evolution.md [hunk 3]
```diff
@@ -126,31 +126,34 @@ Since then, B.1.1.7 has spread across the UK and internationally, and it has now
...
 B.1.351 contains several mutations at the RBD including K417N, E484K, and N501Y.
 While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640] due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
+Additionally, an analysis of a pseudovirus expressing the 501Y.V2 spike protein (B.1.351) showed that this variant demonstrates increased resistance to neutralization by convalescent plasma, even though total binding activity remained mostly intact [@doi:10.1038/s41591-021-01285-x].
+Further, using a live virus neutralization assay (LVNA), it was shown that 501Y.V2 (B.1.351) is poorly neutralized by plasma from individuals who responded to non-501Y.V2 variants, whereas 501Y.V2 infection-elicited plasma was able to cross-neutralize earlier, non-501Y.V2 variants, suggesting that vaccines targeting VOCs may be effective against other mutant lineages [@doi:10.1038/s41586-021-03471-w].
```
Comment (agitter) [inline content/09.evolution.md:139]: This long sentence could be split into two

File: content/09.evolution.md [hunk 4]
```diff
@@ -168,7 +171,9 @@ Moderna recently published data for their mRNA-1273 vaccine that showed no signi
...
-For now, the consensus appears to be that the FDA approved vaccines still seem to be effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a slightly lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948]. 
+In a study focusing on serum samples from a combination of convalescent individuals, those who obtained the mRNA-1273 vaccine, and those who obtained Novavax, in comparison to the D614G variant, the B.1.419 variant was 2-3 times less sensitive to neutralization while the B.1.351 variant was 9-14 times less sensitive [@doi:10.1056/NEJMc2103740].
+Indeed, the E484K substitution seen in the P.1 and B.1.315 variants of the B.1.1.7 lineage are broadly reported to substantially reduce the efficacy of mRNA-based vaccines [@doi:10.1016/S2666-5247(21)00068-9;10.1038/s41586-021-03412-7;10.1056/NEJMc2103740].
```
Comment (agitter) [inline content/09.evolution.md:175]:
```suggestion
Indeed, the E484K substitution seen in the P.1 and B.1.315 variants of the B.1.1.7 lineage are broadly reported to substantially reduce the efficacy of mRNA-based vaccines [@doi:10/f53t; @doi:10.1038/s41586-021-03412-7; @doi:10.1056/NEJMc2103740]. 
```
The short doi is needed when we have special characters like `(` in the doi.

File: content/metadata.yaml [hunk 5]
```diff
@@ -1162,3 +1162,22 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2021-04-05
+ -
```
Comment (agitter) [inline content/metadata.yaml:1165]: The content looks good, but something is off in the yaml file formatting.  I'll check it out and push a commit to change anything I find.

**Review by agitter — CHANGES_REQUESTED** [review]
Thanks for the contributions.  This looks good to me in general.  I have some minor comments and reference updates.

File: content/09.evolution.md (hunk 3 above)
Comment (yusha-sun) [reply content/09.evolution.md]: Done in latest commit!

**Comment by yusha-sun** [comment]
@agitter Thanks, and made changes as suggested!

**Review by agitter — APPROVED** [review]
Thanks @yusha-sun.  This looks good to me now, so I'm approving.

I won't merge yet so we can check the references in the latest manuscript build and see whether anyone with more expertise about these variants has other comments.

File: content/09.evolution.md [hunk 6]
```diff
@@ -126,31 +126,34 @@ Since then, B.1.1.7 has spread across the UK and internationally, and it has now
...
 However, the spotlight has been cast on other variants of unknown clinical relevance due to the increase of cases observed that have been associated with B.1.1.7 in particular.  
-Although early in its ascendency, B.1.427/429 is SARS-CoV-2 variants that was detected in California, USA and also known as CAL.20C [@doi:10.1101/2021.01.18.21249786].
+Although early in its ascendency, B.1.427/429 are SARS-CoV-2 variants that was detected in California, USA and also known as CAL.20C [@doi:10.1101/2021.01.18.21249786].
 It was first detected in July 2020 but was not detected again until October 2020. 
 In December 2020, B.1.427/429 accounted for ~24% of the total cases in Southern California and ~36% of total cases in the Los Angeles area. 
-B.1.427/429 has now been detected in New York and Washington, D.C., and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
+B.1.427/429 have now been detected in New York and Washington, D.C., and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
```
Comment (RLordan) [inline content/09.evolution.md:153]:
```suggestion
B.1.427/429 have now been detected in several U.S. states and at least 38 countries worldwide [@url:https://www.gisaid.org/hcov19-variants/; @doi:10.1101/2021.01.18.21249786].
```

Comment (RLordan) [reply content/09.evolution.md:153]: @rando I think you had mentioned a phylogenetic tree potentially, there seems to be some available with Gisaid
https://www.gisaid.org/hcov19-variants/

File: content/09.evolution.md [hunk 7]
```diff
@@ -168,7 +171,9 @@ Moderna recently published data for their mRNA-1273 vaccine that showed no signi
...
+In a study focusing on serum samples from a combination of convalescent individuals, those who obtained the mRNA-1273 vaccine, and those who obtained Novavax, in comparison to the D614G variant, the B.1.419 variant was 2-3 times less sensitive to neutralization while the B.1.351 variant was 9-14 times less sensitive [@doi:10.1056/NEJMc2103740].
+Indeed, the E484K substitution seen in the P.1 and B.1.315 variants of the B.1.1.7 lineage are broadly reported to substantially reduce the efficacy of mRNA-based vaccines [@doi:10/f53t; @doi:10.1038/s41586-021-03412-7; @doi:10.1056/NEJMc2103740]. 
+For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally to modestly effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948], though select VOCs may present challenges.
```
Comment (RLordan) [inline content/09.evolution.md:176]:
```suggestion
For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948], though select VOCs may present challenges. 
```

Comment (RLordan) [reply content/09.evolution.md:176]: I picked generally over modestly as its probably too early to quantify the effectiveness

**Review by RLordan — COMMENTED** [review]
Welcome @yusha-sun, some minor additions from me.

**Comment by yusha-sun** [comment]
@RLordan Thank you, made changes!

**Comment by rando2** [comment]
Looks great! Welcome @yusha-sun!

**Pull request merged**